papermill/tests/test_translators.py

```diff
@@ -238,6 +238,62 @@ def test_translate_comment_julia(test_input, expected):
     assert translators.JuliaTranslator.comment(test_input) == expected
 
 
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        ("foo", '"foo"'),
+        ('{"foo": "bar"}', '"{""foo"": ""bar""}"'),
+        ({"foo": "bar"}, 'containers.Map({\'foo\'}, {"bar"})'),
+        ({"foo": '"bar"'}, 'containers.Map({\'foo\'}, {"""bar"""})'),
+        ({"foo": ["bar"]}, 'containers.Map({\'foo\'}, {{"bar"}})'),
```

Comment:
Let's add a test for `{1: "foobar"}` to make sure the translator does something sane (I think it should translate the 1 to a string if the matlab Map can't support it as an integer).

Reply:
Same for `{1.0: "foobar"}`, {None, "foobar"}, and "{True: "foobar"} since Matlab has tighter type constraints here than other languages.

Reply:
Good point. I have added these tests. They pass, but you might want to review if the current implementation displays the behavior that you would expect. Currently, it simply makes a char-array out of everything, so 1.0 becomes '1.0', None becomes 'None' and True becomes 'True'.

Reply:
That's reasonable. Thanks for adding the tests